content/docs/how-to-guides/import-an-existing-database.md

```diff
@@ -4,28 +4,68 @@ redirectFrom:
   - /docs/cloud/tutorials
 ---
 
-Importing an existing PostgreSQL database to Neon can be done in almost the same way
-as copying a database from one server to another. PostgreSQL provides several import methods.
-The most efficient and straightforward method is using the `pg_dump` utility.
+This topic describes how to import an existing PostgreSQL database to Neon. The instructions can also be used to migrate a database from one Neon project to another. For example, you can use the instructions to migrate a database from a Neon project created with PostgreSQL 14 to a Neon project created with PostgreSQL 15.
+
+PostgreSQL provides several import methods. The most efficient method is using the `pg_dump` utility with connection strings, as shown:
```

Comment:
> The most efficient

In most (but not all) cases import of a physical backup would work faster. May be just call it "One of methods"

Reply:
Revised.